src/main/java/edu/rpi/legup/puzzle/skyscrapers/SkyscrapersClue.java

```diff
@@ -2,6 +2,8 @@
 
 import edu.rpi.legup.model.gameboard.PuzzleElement;
 
+import java.awt.*;
```

Comment:
Please refrain from importing whole packages, and only import what you need. Importing more than necessary will affect the time needed to compile and build the project.

Reply:
removed